packages/gafl-webapp-service/src/pages/name/route.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+import pageRoute from '../../page-route.js'
+import Joi from '@hapi/joi'
+
+export default pageRoute(
+  'name',
+  '/name',
```

Comment:
Can we use /buy/name - need to keep URL's as consistent with the existing service as possible.

Reply:
done